package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "openmrs-owa-labworkflow",
+  "version": "0.1.0",
+  "description": "The is used to manage the workflow of lab orders",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/openmrs/openmrs-owa-labworkflow.git"
+  },
+  "dependencies": {
+    "@openmrs/react-components": "^0.2.0-SNAPSHOT.8",
+    "babel-polyfill": "^6.26.0",
+    "react": "^15.4.1",
+    "react-dom": "^15.4.1",
+    "react-redux": "^5.0.1",
+    "react-router": "^3.0.0",
+    "react-router-dom": "^4.3.1",
+    "redux": "^3.6.0",
+    "redux-form": "^7.4.2",
+    "redux-promise-middleware": "^4.2.0",
+    "redux-saga": "^0.16.0",
+    "redux-thunk": "^2.1.0"
```

Comment:
If we do decide not to use Thunk and Promise-Middleware, let's just remember to remove these dependencies.